### PR TITLE
shared.cfg: update ovmf boot code file path

### PR DESCRIPTION
**Pull request by xutian**
ID: 1370354

Signed-off-by: Xu Tian xutian@redhat.com

### CODE REVIEW

File: shared/cfg/guest-hw.cfg [hunk 1]
```diff
@@ -356,7 +356,7 @@ variants:
         no Win2003 WinXP WinVista
         # Please update this based on your test env
         ovmf_path = /usr/share/OVMF
-        ovmf_code_filename = OVMF_CODE.fd
+        ovmf_code_filename = OVMF_CODE_secboot.fd
```
Comment (CongLi) [inline shared/cfg/guest-hw.cfg]: Should be "OVMF_CODE.secboot.fd"

**Comment by xutian** [comment]
@CongLi  re-correct it, thanks your comments

**Comment by xutian** [comment]
xutian@dhcp-10-169 ~/Devel/avocado-vt/virttest (ovmf_fix) $ rpm -qpl ~/Downloads/OVMF-20160608b-1.git988715a.el7.noarch.rpm 
/usr/share/OVMF
/usr/share/OVMF/OVMF_CODE.secboot.fd
/usr/share/OVMF/OVMF_VARS.fd
/usr/share/OVMF/UefiShell.iso
/usr/share/doc/OVMF/Licenses
/usr/share/doc/OVMF/Licenses/CryptoPkg-License.txt
/usr/share/doc/OVMF/Licenses/FatPkg-License.txt
/usr/share/doc/OVMF/Licenses/IntelFrameworkModulePkg-License.txt
/usr/share/doc/OVMF/Licenses/MdeModulePkg-License.txt
/usr/share/doc/OVMF/Licenses/MdePkg-License.txt
/usr/share/doc/OVMF/Licenses/OpensslLib-License.txt
/usr/share/doc/OVMF/Licenses/OptionRomPkg-License.txt
/usr/share/doc/OVMF/Licenses/OvmfPkg-License.txt
/usr/share/doc/OVMF/Licenses/PcAtChipsetPkg-License.txt
/usr/share/doc/OVMF/Licenses/SecurityPkg-License.txt
/usr/share/doc/OVMF/Licenses/ShellPkg-License.txt
/usr/share/doc/OVMF/Licenses/UefiCpuPkg-License.txt
/usr/share/doc/OVMF/README
/usr/share/doc/OVMF/ovmf-whitepaper-c770f8c.txt

**Comment by xutian** [comment]
merged, thanks @CongLi

**Pull request merged**